website/versioned_docs/version-pre-2.0/conceptual/node/extrinsics.md

```diff
@@ -94,5 +94,5 @@ protection logic that is needed by the transaction pool.
 
 - [Reference Documentation](https://substrate.dev/rustdocs/master/sp_runtime/traits/trait.Extrinsic.html)
 - [Runtime Execution](development/module/execution.md)
-- [Transaction Fees](development/module/fees.md)
+<!-- TODO - [Transaction Fees](development/module/fees.md) -->
```

Comment:
Why are these inks being removed? https://substrate.dev/docs/en/development/module/fees still exists

Reply:
There was an incorrect link in an old doc. I'll add them again.